### PR TITLE
Repokid JSON logging support

### PR DESCRIPTION
**Pull request by castrapel**

### CODE REVIEW

File: repokid/__init__.py [hunk 1]
```diff
@@ -11,15 +11,30 @@
...
 import os
+import socket
+import sys
 
 import import_string
+import logmatic
+from pytz import timezone
 
-__version__ = "0.10.0"
+__version__ = "0.10.1"
```
Comment (mcpeak) [inline repokid/__init__.py]: I think we want a minor version update for log format changing

File: repokid/__init__.py [hunk 2]
```diff
@@ -60,8 +75,10 @@ def init_logging():
     Returns:
         None
     """
+    logging_format = "%(asctime)s - %(levelname)s - %(name)s - [%(filename)s:%(lineno)s - %(funcName)s() ] - %(message)s"  # noqa: E501
+
     if CONFIG:
-        logging.config.dictConfig(CONFIG["logging"])
+        logging_format = CONFIG["logging"]
```
Comment (mcpeak) [inline repokid/__init__.py]: does this work?

Comment (castrapel) [reply repokid/__init__.py]: Yes, we later do a `logging.basicConfig(level="DEBUG", format=logging_format)`. With the previous version, logs are duplicated.

File: repokid/__init__.py (hunk 1 above)
Comment (castrapel) [reply repokid/__init__.py]: Oops, fixed thanks

**Comment by mcpeak** [comment]
Let's test and then merge. Looks good, thank you.

**Pull request merged**